pulp_rpm/app/viewsets.py

```diff
@@ -1273,4 +1274,8 @@ def refresh(self, request, pk):
 
         # Update TaskGroup that all child task are dispatched
         task_group.finish()
+
+        acs.last_refreshed = now()
+        acs.save()
```

Comment:
So this is fine, but do you think the field in pulpcore ought to be `auto_now` so that it automatically refreshes on save?  Would that make sense?  Or would you sometimes want to save the ACS without updating this field.

https://github.com/pulp/pulpcore/blob/main/pulpcore/app/models/base.py#L62=

Reply:
Of course, the best example to update is paths.